diff --git a/tcl/pd_deken.tcl b/tcl/pd_deken.tcl
index 13bbdabb..216ccd91 100644
--- a/tcl/pd_deken.tcl
+++ b/tcl/pd_deken.tcl
@@ -28,10 +28,10 @@ package require Tcl 8.4 9
 
 package require http 2
 # try enabling https if possible
-if { [catch {package require tls 1.7} ] } {} else {
+if { [catch {package require tls 1.7-} ] } {} else {
     ::tls::init \
         -autoservername 1 \
-        -ssl2 false -ssl3 false -tls1 true
+        -tls1 false -tls1.1 false
     ::http::register https 443 ::tls::socket
 }
 # try enabling PROXY support if possible
